content/20.results.md

```diff
@@ -63,4 +63,14 @@ The proportion of Hispanic authors and honorees at these venues may also be infl
 
 ### Predicting Nationality with LSTM Neural Networks and Wikipedia
 
+We achieved a high overall accuracy on this test data with similar precision-recall values over the classes as we saw for our Wiki2019 dataset’s test set, giving confidence in our model.
```

Comment:
I'm guessing we'll revisit this section and the one below once figures are together.